qcfractal/tests/test_collections.py

```diff
@@ -732,10 +745,56 @@ def test_compute_reactiondataset_keywords(fractal_compute_server):
     assert kw.values["scf_type"] == "df"
 
 
+def live_fractal_or_skip():
+    """Ensure Fractal live connection can be made"""
+    import requests
+    try:
+        requests.get('https://api.qcarchive.molssi.org:443', json={}, timeout=5)
+        return ptl.FractalClient()
+    except (requests.exceptions.ConnectionError, ConnectionRefusedError):
+        return pytest.skip("Could not make a connection to central Fractal server")
```

Comment:
It might be good to add this to `testing.py` as it will be useful in other places.